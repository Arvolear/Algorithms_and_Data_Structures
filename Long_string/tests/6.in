4943783635750236103058198193765
79949232809042672980919748044701698