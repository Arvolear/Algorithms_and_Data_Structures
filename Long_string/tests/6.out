79954176592678423217022806242895463
79944289025406922744816689846507933
395251708852131047564876536712011888884208199349623423301828512970
16171